npm/private/npm_translate_lock.bzl

```diff
@@ -146,6 +146,16 @@ _ATTRS = {
         }
         """,
     ),
+    "lifecycle_hooks_execution_requirements": attr.string_list_dict(
+        doc = """Execution requirements applied to the preinstall, install and postinstall lifecycle hooks on npm packages.
+        The execution requirements can be defined per package by package name or globally using "*".
+        For example:
+        lifecycle_hooks_execution_requirements: {
+            "*": ["requires_network"],
```

Comment:
`requires-network` and `no-sandbox` are the values bazel expects right?

Reply:
In my understanding, the use case is here https://github.com/aspect-build/rules_js/issues/270, so the user run 
`bazel --nosandbox_default_allow_network` but still want `npm_import` use network, so by setting the `execution_requirements`, the user can run `bazel --nosandbox_default_allow_network`  and the `npm_import` can still access network for the specified packages.

Reply:
Oh. I was referring to the `_` vs the `_`. I believe the values should have `-` to be correct.

Reply:
Oh, yeah, you are right, thanks.